Assignment1-JavaScript/palindrome.js

```diff
@@ -0,0 +1,70 @@
+
+function LinkedList(val){
+    this.val = val;
+    this.next = null;
+}
+
+function isPalindrome(x) {
+    if (!x || !Headers.next){
```

Comment:
Not sure what you are trying to do with Headers.next here since thats an HTTP fetch request API object. I'm gonna imagine that is for checking if the list passed in has a next field.